src/DynamoRevit/DynamoRevit.cs

```diff
@@ -374,12 +374,30 @@ public static string GetGeometryFactoryPath(string corePath)
                 throw new FileNotFoundException("File not found", dynamoAsmPath);
 
             var utilities = assembly.GetType("DynamoShapeManager.Utilities");
-            var getGeometryFactoryPath = utilities.GetMethod("GetGeometryFactoryPath");
+            var getGeometryFactoryPath = utilities.GetMethod("GetGeometryFactoryPath2");
 
+            //if old method is called default to 224.4.0
             return (getGeometryFactoryPath.Invoke(null,
-                new object[] { corePath, Versions.ShapeManager }) as string);
+                new object[] { corePath, new Version(224,4,0)  }) as string);
```

Comment:
What is the rationale behind falling back to version 224.4 even for older Revit branches?

Reply:
no rationale! I mistake in cherry picking. Thanks for catching it.